Source: William Orr <will@worrbase.com>
Upstream: https://github.com/gugod/App-perlbrew/pull/288
Reason: Fixes bashrc issues
---
 lib/App/perlbrew.pm | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/lib/App/perlbrew.pm b/lib/App/perlbrew.pm
index 48f1888..4117137 100644
--- a/lib/App/perlbrew.pm
+++ b/lib/App/perlbrew.pm
@@ -2042,7 +2042,7 @@ __perlbrew_set_path () {
     fi
     unset MANPATH_WITHOUT_PERLBREW
 
-    PATH_WITHOUT_PERLBREW=`$perlbrew_command display-pristine-path`
+    PATH_WITHOUT_PERLBREW=$(eval $perlbrew_command display-pristine-path)
     if [ -n "$PERLBREW_PATH" ]; then
         export PATH=${PERLBREW_PATH}:${PATH_WITHOUT_PERLBREW}
     else
@@ -2058,9 +2058,9 @@ __perlbrew_activate() {
 
     if [[ -n "$PERLBREW_PERL" ]]; then
         if [[ -z "$PERLBREW_LIB" ]]; then
-            eval "$($perlbrew_command env $PERLBREW_PERL)"
+            $(eval $perlbrew_command env $PERLBREW_PERL)
         else
-            eval "$(${perlbrew_command} env $PERLBREW_PERL@$PERLBREW_LIB)"
+            $(eval $perlbrew_command env $PERLBREW_PERL@$PERLBREW_LIB)
         fi
     fi
 
@@ -2068,7 +2068,7 @@ __perlbrew_activate() {
 }
 
 __perlbrew_deactivate() {
-    eval "$($perlbrew_command env)"
+    $(eval $perlbrew_command env)
     unset PERLBREW_PERL
     unset PERLBREW_LIB
     __perlbrew_set_path
-- 
1.8.1.3

